### DISCUSSION TITLE
Mitosis support?

### DISCUSSION THREAD

**Discussion by b-barry** [chakra-ui/zag#102] [General]
# 🚀 Feature request

> Please describe your request in one or two sentences.
Hello,

Do you plan to support [Mitosis](https://github.com/BuilderIO/mitosis)? It share a lot similarity with Solid.js

Thanks for the good library.

**Comment by segunadebayo** [comment]
Hi @b-barry,

Open to seeing how this evolves over time. We don't have plans to support Mitosis but we're open to collaboration to make it happen.

**Comment by segunadebayo** [comment]
Closing as outdated — no plans for Mitosis integration.
